Build fails with USE_GPL_LIBS=0 again

On Ubuntu 14.04.3, my last successful build with JULIA_THREADS=1 was 0a2a8b0.
`make clean` and `make cleanall` did not help.  The build succeeds without JULIA_THREADS=1.

```
deprecated.jl
error during bootstrap:
LoadError(at "sysimg.jl" line 319: LoadError(at "deprecated.jl" line 510: UndefVarError(var=:FFTW)))
rec_backtrace at <invalid>:0
record_backtrace at <invalid>:0
jl_throw at <invalid>:0
jl_undefined_var_error at <invalid>:0
unknown function (ip: 0x2b2a44042425)
jl_apply_generic at <invalid>:0
do_call at <invalid>:0
eval at <invalid>:0
do_call at <invalid>:0
eval at <invalid>:0
do_call at <invalid>:0
eval at <invalid>:0
do_call at <invalid>:0
eval at <invalid>:0
eval at <invalid>:0
eval_body at <invalid>:0
jl_toplevel_eval_body at <invalid>:0
jl_toplevel_eval_flex at <invalid>:0
jl_toplevel_eval_flex at <invalid>:0
jl_parse_eval_all at <invalid>:0
jl_load at <invalid>:0
unknown function (ip: 0x2b2c489eb013)
jl_apply_generic at <invalid>:0
unknown function (ip: 0x2b2c4964413f)
jl_apply_generic at <invalid>:0
do_call at <invalid>:0
eval at <invalid>:0
jl_toplevel_eval_flex at <invalid>:0
jl_eval_module_expr at <invalid>:0
jl_toplevel_eval_flex at <invalid>:0
jl_parse_eval_all at <invalid>:0
jl_load at <invalid>:0
unknown function (ip: 0x401bf8)
unknown function (ip: 0x40217a)
unknown function (ip: 0x40186b)
__libc_start_main at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
unknown function (ip: 0x4018b1)
unknown function (ip: (nil))

*** This error is usually fixed by running `make clean`. If the error persists, try `make cleanall`. ***
make[1]: *** [/home/sachs/src/julia-multithreading/usr/lib/julia/sys.o] Error 1
make: *** [julia-sysimg-release] Error 2```

--- a/base/deprecated.jl
+++ b/base/deprecated.jl
@@ -507,10 +507,12 @@ export float32_isvalid, float64_isvalid
 
 # 12087
 @deprecate call(P::Base.DFT.ScaledPlan, A) P * A
-@deprecate call(P::Base.FFTW.DCTPlan, A) P * A
-@deprecate call(P::Base.FFTW.cFFTWPlan, A) P * A
-@deprecate call(P::Base.FFTW.rFFTWPlan, A) P * A
-@deprecate call(P::Base.FFTW.r2rFFTWPlan, A) P * A
+if Base.USE_GPL_LIBS
+    @deprecate call(P::Base.FFTW.DCTPlan, A) P * A
+    @deprecate call(P::Base.FFTW.cFFTWPlan, A) P * A
+    @deprecate call(P::Base.FFTW.rFFTWPlan, A) P * A
+    @deprecate call(P::Base.FFTW.r2rFFTWPlan, A) P * A
+end
 for f in (:plan_fft, :plan_ifft, :plan_bfft, :plan_fft!, :plan_ifft!, :plan_bfft!, :plan_rfft)
     @eval @deprecate $f(A, dims, flags) $f(A, dims; flags=flags)
     @eval @deprecate $f(A, dims, flags, tlim) $f(A, dims; flags=flags, timelimit=tlim)

